src/Symfony/Component/Security/Http/Authenticator/Passport/Badge/RememberMeBadge.php

```diff
@@ -30,6 +25,36 @@
  */
 class RememberMeBadge implements BadgeInterface
 {
+    private $enabled = false;
+
+    /**
+     * Enables remember me cookie creation.
+     *
+     * In most cases, you should not use this in your authenticator, but
+     * instead rely on {@see RememberMeConditionsListener} to enable it
+     * for you.
```

Comment:
While this is true, it IS perfectly valid to call this in your authenticator to explicitly force remember me. I don't want to overly discourage people from this :)

Reply:
I did want to discourage people from doing this :) But after reading the referenced Oauth2 issue, I agree that we shouldn't overly discourage using this. I've updated the sentence